src/Data/Nat/Properties.agda

```diff
@@ -924,3 +933,170 @@ im≡jm+n⇒[i∸j]m≡n i j m n eq = begin
 ⌊n/2⌋≤′n : ∀ n → ⌊ n /2⌋ ≤′ n
 ⌊n/2⌋≤′n zero    = ≤′-refl
 ⌊n/2⌋≤′n (suc n) = ≤′-step (⌈n/2⌉≤′n n)
+
+
+
+
+
+-- ***************************************************************************
+-- Auxiliary items needed for the Bin items of proposal by S.M.
+-- Some of them may have another usage.
+
+natDTO       = ≤-decTotalOrder
+natSTO       = strictTotalOrder
+natDecSetoid = DecTotalOrder.Eq.decSetoid natDTO    -- for ℕ
+natEquiv     = DecSetoid.isEquivalence natDecSetoid
+
```

Comment:
Again, don't perform renaming in public scope in the standard library. Either perform renaming in the file where you are using them or use the original names directly in your proofs.